src/ert/scheduler/lsf_driver.py

```diff
@@ -407,6 +411,7 @@ async def _process_job_update(self, job_id: str, new_state: AnyJob) -> None:
 
         if event:
             if isinstance(event, FinishedEvent):
+                await self._dump_bhist_job_summary_to_runpath(job_id)
```

Comment:
If this takes some time (flaky ssh and stuff), then the driver will continue to poll for the job in the meantime (and if potentially bjobs will fallback to bhist as well). Can that make some problems? Should we potentially move the `del` statements prior to the bhist call?

Reply:
Yes, good idea!